rule-packs/trellix-endpoint-security.json

```diff
@@ -0,0 +1,110 @@
+[
+    {
+        "name": "trellix-threats-1",
+        "description": "This query will return threats with an unresolved status",
+        "queries": [
+            {
+                "name": "query0",
+                "query": "FIND trellix_threat with status = unresolved",
```

Comment:
can we make all of the J1QL keywords in caps?

Reply:
Done